src/components/PopUp/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import SimpleBackdrop from "./Backdrop/Backdrop";
+
+function PopUp(props) {
+  return (
+    <>
+      <SimpleBackdrop></SimpleBackdrop>
```

Comment:
The name of the file is Backdrop.js, but the name of the component in this file is SimpleBackDrop this is confusing